### DISCUSSION TITLE
Nsawg build step not working on build on mac

### DISCUSSION THREAD

**Discussion by GeorgiNik** [jasontaylordev/CleanArchitecture#501] [General]
The build fails on mac and linux on the nswag generation step. Is there a way to make this work on mac and linux?

**Comment by Manuman76** [comment]
Having the same issue, dotnet 6.0.102 on Linux.
Dotnet Run failed with:

`/home/manuel/sources/CleanProject/src/WebUI/WebUI.csproj(59,9): error MSB3073: The command "dotnet "/home/manuel/.nuget/packages/nswag.msbuild/13.9.4/build/../tools/Net50/dotnet-nswag.dll" run nswag.json /variables:Configuration=Debug" exited with code 150.`

**Comment by deceptiveSimplicity** [comment]
Are you using the latest template code? It seems like in your project $(NSwagExe_Net50) is used instead of $(NSwagExe_Net60).
https://github.com/jasontaylordev/CleanArchitecture/blob/35b490110f699c2ba427fd6b49e98babc16390c0/src/WebUI/WebUI.csproj#L59

**Reply by Manuman76** [reply]
thanks for the answer... I have the last template code, still referencing net50, even when I update to net60, i get the same error. My guess is that nswag toolchain is different on Mac/Linux. The template should not depend on this... make it hard to integrate in a linux CI/CD pipeline.

Anyone have a solution to this ?

**Reply by Manuman76** [reply]
OK, I see... the nuget package is not updated... when using the use as a template from github, everything is good...! ciao!
